ui/threejs/moveToFit.js

```diff
@@ -68,6 +68,8 @@ export const moveToFit = ({
     return;
   }
 
+   controls.reset();
```

Comment:
This resets the camera to point back at the center if it has been panned. This hasn't been an issue since we are creating a new orbitView with each view, but when using updateGeometry the camera would zoom to fit everything but be pointed at the wrong spot.